includes/base/controls-stack.php

```diff
@@ -998,6 +1010,14 @@ final public function get_config() {
 			}
 		}
 
+		foreach ( $this->additional_config as $key => $value ) {
+			if ( isset( $this->config[ $key ] ) && is_array( $this->config[ $key ] ) && is_array( $value ) ) {
+				$this->config[ $key ] = array_merge( $this->config[ $key ], $value );
+			} else {
+				$this->config[ $key ] = $value;
+			}
+		}
+
```

Comment:
Not sure if it fits here, but maybe you can use [wp_parse_args()](https://developer.wordpress.org/reference/functions/wp_parse_args/)?

Reply:
Good instinct!
Just gave it a shot and it works :)

Reply:
🔥